src/main/java/org/opentripplanner/middleware/tripMonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -257,133 +263,199 @@ private void enqueueNotification(TripMonitorNotification ...tripMonitorNotificat
     }
 
     /**
-     * Check whether monitored trip check should be skipped.
-     * TODO: Should this be a method on {@link MonitoredTrip}?
+     * Determine whether to skip checking the monitored trip at this instant. The decision on whether to skip the check
+     * takes into account the current time, the lead time prior to the itinerary start and the last time that the trip
+     * was checked. Skipping the check should only occur if the previous trip has ended and the next trip meets the
+     * following criteria for skipping a check:
+     *
+     * - the current time is before when the lead time before the next itinerary starts
+     * - the current time is after the lead time before the next itinerary starts, but is over an hour until the
+     *     itinerary start time and the trip has already been checked within the last 60 minutes
+     * - the current time is after the lead time before the next itinerary starts and between 60-15 minutes prior to the
+     *     itinerary start time, but a check has occurred within the last 15 minutes
```

Comment:
This feels a bit wordy. The language could be simplified to improve how it reads (without losing clarity).

Reply:
I updated this a little bit in 6fd00a976c063c5b46e7254a995fbbb628b90445, but prefer to be hyper-specific when talking about timing matters.